app.js

```diff
@@ -84,6 +84,21 @@ app.use(compression())
 app.locals.GITHUB_SHA = process.env.GITHUB_SHA || null
 app.locals.hasData = hasData
 
+/**
+ * Create an asset path helper for templates
+ * If a CDN_PREFIX is set in env, and mode is production,
+ * the helper will return the path with the CDN prefix,
+ * otherwise it just returns the path
+ */
+app.locals.asset = (path) => {
+  const cdnprefix = process.env.CDN_PREFIX || '';
+
+  if (process.env.NODE_ENV === 'production') {
+    return cdnprefix + path;
+  }
```

Comment:
Would we ever want to use a CDN not in production mode?

Reply:
I don't think so .... but I suppose I could change this to check for the existence of the env var, so you could just add it to whatever environments you want to load assets from the cdn?

Reply:
No your right that was a foolish question we would want to be compile our assets as we go.